flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/drivers/parameter/LongParameter.java

```diff
@@ -52,16 +52,6 @@ public LongParameter(ParameterizedBase owner, String name) {
 	public LongParameter setDefaultValue(long defaultValue) {
 		super.setDefaultValue(defaultValue);
 
-		if (hasMinimumValue) {
-			Util.checkParameter(defaultValue >= minimumValue,
-				"Default value (" + defaultValue + ") must be greater than or equal to minimum (" + minimumValue + ")");
-		}
-
-		if (hasMaximumValue) {
-			Util.checkParameter(defaultValue <= maximumValue,
-				"Default value (" + defaultValue + ") must be less than or equal to maximum (" + maximumValue + ")");
-		}
-
```

Comment:
ok, should we make this explicit in the `LongParamater` class' javadoc? For example:
```
/**
 * A {@link Parameter} storing a {@link Long} within <tt>min</tt> and <tt>max</tt> bounds (inclusive).
 *
 * <p>Note that the default value may be outside of these bounds.
 */
```